### PR TITLE
fix-condition

### PR DESCRIPTION
**Pull request by Umangvaghela**
fix-condition

### CODE REVIEW

File: includes/class-wc-form-handler.php [hunk 1]
```diff
@@ -351,7 +351,7 @@ public static function pay_action() {
 					$available_gateways[ $payment_method ]->validate_fields();
 
 					// Process
-					if ( wc_notice_count( 'error' ) == 0 ) {
+					if ( 0 ===  wc_notice_count( 'error' )  ) {
```
Comment (mikejolley) [inline includes/class-wc-form-handler.php]: There is extra white space

File: includes/class-wc-form-handler.php [hunk 2]
```diff
@@ -747,12 +747,12 @@ public static function add_to_cart_action( $url = false ) {
 		}
 
 		// If we added the product to the cart we can now optionally do a redirect.
-		if ( $was_added_to_cart && wc_notice_count( 'error' ) === 0 ) {
+		if ( $was_added_to_cart && 0 ===  wc_notice_count( 'error' )  ) {
```
Comment (mikejolley) [inline includes/class-wc-form-handler.php]: There is extra white space

File: includes/class-wc-form-handler.php [hunk 3]
```diff
@@ -747,12 +747,12 @@ public static function add_to_cart_action( $url = false ) {
...
 			// If has custom URL redirect there
 			if ( $url = apply_filters( 'woocommerce_add_to_cart_redirect', $url ) ) {
 				wp_safe_redirect( $url );
 				exit;
-			} elseif ( get_option( 'woocommerce_cart_redirect_after_add' ) === 'yes' ) {
+			} elseif ( 'yes' ===  get_option( 'woocommerce_cart_redirect_after_add' )  ) {
```
Comment (mikejolley) [inline includes/class-wc-form-handler.php]: There is extra white space

**Review by mikejolley — CHANGES_REQUESTED** [review]

File: includes/class-wc-form-handler.php (hunk 3 above)
Comment (Umangvaghela) [inline includes/class-wc-form-handler.php]: @mikejolley

Fix this issue on new pull request https://github.com/woocommerce/woocommerce/pull/16558

**Pull request closed without merging**